src/platform/packages/shared/kbn-scout/src/playwright/page_objects/painless_lab_page.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { ScoutPage } from '..';
+
+const DEFAULT_LOADING_TIMEOUT = 10_000;
```

Comment:
We have default timeouts set in Playwright default configuration:
https://github.com/fake-haris/kibana/blob/painless-lab-scout/src/platform/packages/shared/kbn-scout/src/playwright/config/create_config.ts#L74-L76

Basically any playwright action by default is limited to 10 seconds and you don't need to pass timeout unless you want to increase or decrease the defaults.

Reply:
you still have to define what you are waiting for: probably `'visible'` ?